README.md

```diff
@@ -146,7 +146,7 @@ message Transaction {
 
 ### Ecosystem
 
-- [ProtoValidate](https://github.com/bufbuild/protovalidate) core repository
+- [`protovalidate`](https://github.com/bufbuild/protovalidate) core repository
```

Comment:
The canonical name is `protovalidate` (not `ProtoValidate` or `Protovalidate`).

It would be great to make the code reflect this naming convention too.